hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -405,6 +405,28 @@ The following diagram shows how the flash read pipeline timing works.
 In this example, the first two host requests trigger a full sequence.
 The third host requests immediately hits in the local cache and responds in order after the first two.
 
+### Flash Buffer
+
+The flash buffer is a small read-only memory that holds multiple entries of recently read flash words.
+This is needed when the flash word is wider than a bus word.
+The flash access time is amortized across the the entire flash word if software accesses in a mostly
+linear sequence.
+
+The flash buffer has a round robin replacement policy when more flash words are read.
+When an erase / program is issued to the flash, the entries are evicted to ensure new words are fetched.
```

Comment:
This is an unconditional eviction, i.e., not based on which address we erase, program, right?

Reply:
let me clarify that, it is conditional.